server/auth/is-authorized.js

```diff
@@ -0,0 +1,11 @@
+const createError = require("http-errors");
+
+const isAuthorized = (req, res, next) => {
+  const yes = req.user._id.toString() === req.params.userId;
```

Comment:
You should validate JWT token

Reply:
So, i have two middlewares named _is-signed-in_ and _is-authorized_ . _is-signed-in_ checks if the user is signed in and _is-authorized_  checks if the user is trying to CRUD his/her own data. If he/she is trying to modify some other users' data, then the request will be invalidated

Reply:
Why do we need it? According to our current project we don't have any roles and policies, it means that each user can rights for CRUD (sure only for his tasks and projects)

Reply:
Is that a big of a deal? Should i delete it and rewrite my code?